README.md

```diff
@@ -10,6 +10,10 @@ The Lumigo OpenTelemetry Distribution for Node.js is made of several upstream Op
 
 **Note:** If you are looking for the Lumigo Node.js tracer for AWS Lambda functions, [`@lumigo/tracer`](https://npm.io/package/@lumigo/tracer) is the package you should use instead.
 
+## Logging support
```

Comment:
I think it's worth adding a link to Lumigo Official Logs documentation as well (same goes for all distros)

Reply:
We don't have it open publicly yet - I'll add it once it's GA